cmd/auth.go

```diff
@@ -0,0 +1,50 @@
+// Copyright (c) The go-boot authors. All Rights Reserved.
+//
+// Use of this source code is governed by the license
+// that can be found in the LICENSE file.
+
+package cmd
+
+import (
+	"fmt"
+	"regexp"
+	"strings"
+
+	"github.com/usbarmory/go-boot/shell"
+	"github.com/usbarmory/go-boot/transparency"
+)
+
+var btConfig transparency.Config
+
+func init() {
+	shell.Add(shell.Cmd{
+		Name:    "bt",
+		Args:    1,
+		Pattern: regexp.MustCompile(`^(?:bt)( none| offline| online)?$`),
+		Syntax:  "(none|offline|online)?",
+		Help:    "show/set boot-transparency status",
```

Comment:
For consistency with `date` command this (and in the README) should be updated to "show/change".